code/game/objects/items/weapons/RCD.dm

```diff
@@ -128,21 +154,20 @@ RCD
 		if(1)
 			mode = 2
 			user << "<span class='notice'>You change RCD's mode to 'Airlock'.</span>"
-			if(prob(20))
-				src.spark_system.start()

```

Comment:
Why are you removing this? I like watching people set themselves on fire while trying to RCD open a floor to drain plasma!